src/Umbraco.Web.UI.Client/src/less/components/umb-form-check.less

```diff
@@ -98,6 +98,7 @@
     .icon,
     .umb-icon {
         font-size: 1.2rem;
+        z-index: 1;
```

Comment:
@nathanwoulfe alternatively just `z-index: 1` to `umb-form-check__icon` or `position: relative` to `ng-include` here: https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/less/components/umb-icon.less#L41

Reply:
@nathanwoulfe maybe we should add this to `umb-form-check__icon` which also set `opacity: 1;`
//cc @nul800sebastiaan